streampark-flink/streampark-flink-kubernetes/src/main/scala/org/apache/streampark/flink/kubernetes/watcher/FlinkJobStatusWatcher.scala

```diff
@@ -234,12 +235,15 @@ class FlinkJobStatusWatcher(conf: JobStatusWatcherConfig = JobStatusWatcherConfi
    * list flink jobs details from rest api
    */
   private def callJobsOverviewsApi(restUrl: String): Option[JobDetails] = {
-    JobDetails.as(
+    logger.info(s"Try to access flink's service via http:${restUrl}/jobs/overview.")
```

Comment:
Is it better to change the log here to debug level?  Job status monitoring is continuous, reducing unnecessary logs as much as possible